Automattic/wp-calypso
# Notifications: Move to apps/

Will come in handy for #32601.

#### Changes proposed in this Pull Request

Notifications: Move to apps/

They seem to be a perfect example of our `apps/` paradigm: 
- Only one (style) `import` of Calypso code AFAICS
- `webpack.config.js` of their own
- Built through `calypso-build`

#### Testing instructions

```
npm run distclean && npm ci
npx lerna run build 
```

Verify that notifications build. Furthermore, follow testing instructions found at PCYsg-elI-p2

```
npm start
```

Verify that Calypso still works. Specifically, smoke-test notifications.
